src/qutip_cupy/__init__.py

```diff
@@ -1 +1,21 @@
 from .version import version as __version__
+
+from qutip.core import data
```

Comment:
We shouldn't be re-exporting parts of `qutip` from this package's main namespace - be sure to `del` it after you've used it.

Reply:
Good point.